.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "AuthProvider"]
```

Comment:
No problem, just made it alot cleaner and provided a way to make it more [extendable](https://github.com/formula1/AuthProvider/tree/master/auth-providers). As much as npm is gigantic mess of applications. I wanted to make sure mine was hackable